composer.json

```diff
@@ -12,7 +12,12 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": "^5.4 | ^7.0"
+        "php": "^5.4 | ^7.0",
+        "dhii/output-renderer-interface": "^0.2",
+        "dhii/output-renderer-abstract": "dev-release/0.1",
+        "dhii/i18n-helper-base": "dev-develop",
+        "dhii/exception-helper-base": "dev-develop",
+        "dhii/stringable-interface": "^0.1"
```

Comment:
Dependency `dhii/stringable-interface` is not used for any invocation - only for type hinting in doc blocks.

Solution: make it a dev dependency.